src/Components/Config/ConfigScreens/AccountsScreen.js

```diff
@@ -145,6 +147,27 @@ class AccountsScreen extends Component {
             </div>
           </section>
         </OnlyIf>
+
+        <section>
+          <h4>LOCK ACCOUNTS</h4>
+          <div className="Row">
+            <div className="RowItem">
+              <div className="Switch">
+                <input
+                  type="checkbox"
+                  name="server.locked"
+                  id="LockAccounts"
+                  checked={this.props.settings.server.locked}
+                  onChange={this.props.handleInputChange}
+                />
+                <label htmlFor="LockAccounts">LOCK ACCOUNTS</label>
+              </div>
+            </div>
+            <div className="RowItem">
+              <p>Created accounts are locked by default.</p>
```

Comment:
This might be slightly confusing to some. Maybe say "If enabled, accounts will be locked on startup."

Reply:
I tried finding something that worked many times haha; I like that better